### DISCUSSION TITLE
coc completion list disappear immediately

### DISCUSSION THREAD

**Discussion by mohamedLT** [neoclide/coc.nvim#4699] [Q&A]
Hello there 
I have been using coc for a while now and just recently a new ls was realesed for v call v-analyzer 
when I set it up with coc it does not show any complition item besides the buf ones and sometimes it popsup and disappear immediately the only time it shows up correctly is when I have a dot before the text like this ` something.foo ` 
I used workspace output and found out that the ls sends the queries when I type but coc does not show them for some reason 
Any idea how to fix this

**Comment by chemzqm** [comment]
Need output from LS https://github.com/neoclide/coc.nvim/wiki/Debug-language-server#using-output-channel
